deploy/crds/mellanox.com_v1alpha1_macvlannetwork_cr.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: mellanox.com/v1alpha1
+kind: MacvlanNetwork
+metadata:
+  name: example-macvlannetwork
+spec:
+#  networkNamespace: ""
+#  master: ""
```

Comment:
can you uncomment `master` and put the same value as in the nic-cluster-policy CR example ?

Reply:
changed